packages/app-mobile/package.json

```diff
@@ -107,6 +107,7 @@
     "babel-loader": "9.1.3",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-plugin-react-native-web": "0.19.12",
+    "fast-deep-equal": "3.1.3",
```

Comment:
`fast-deep-equal` is currently a dependency of the `lib` package. In `lib`, it's used to compare `selectedNoteTags` in `reducer.ts`.